### PR TITLE
Fix local run to use guacamole dependencies jar

### LINKED ISSUE #22: local run example is wrong, uses outdated guacamole-with-dependencies jar
In the readme this run:

```
vcb-guacamole-local \
    base_config.json \
    infrastructure-configs/local.json \
    guacamole-configs/default.json \
    benchmarks/local-wgs1/benchmark.json \
    --guacamole-jar ~/sinai/git/guacamole/target/guacamole-with-dependencies-0.0.1-SNAPSHOT.jar \
    --out-dir results
```

Is actually using an old guacamole jar from before some pom changes. It should actually take both a guacamole jar and a guacamole deps jar, since these are separate jars now.

### PR DESCRIPTION
**Pull request by timodonnell**
Closes #22.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,24 +32,24 @@ Note that variant calls are associated with patients, not samples. This is the "
...
+Change the `--guacamole-jar` and `--guacamole-dependencies-jar` paths below for your environment.
 
 ```
 vcb-guacamole-local \
     base_config.json \
     infrastructure-configs/local.json \
     guacamole-configs/default.json \
     benchmarks/local-wgs1/benchmark.json \
-    --guacamole-jar ~/sinai/git/guacamole/target/guacamole-with-dependencies-0.0.1-SNAPSHOT.jar \
+    --guacamole-jar ~/sinai/git/guacamole/target/guacamole-0.0.1-SNAPSHOT.jar \
```
Comment (ryan-williams) [inline README.md]: maybe we should expose a `$GUAC_HOME` env var API that will find the JARs for you and set up the classpath, like `scripts/guacamole` does

Comment (ryan-williams) [inline README.md]: in any case I think for the documentation here you should sub in such a var, so `$GUAC_HOME/target/guacamole-…` etc.

File: variant_calling_benchmarks/common.py [hunk 2]
```diff
@@ -28,7 +28,10 @@ def extract_loci_string(patient, variant_filenames):
 
 def add_common_run_args(parser):
     parser.add_argument("configs", nargs="+", help="JSON config files")
-    parser.add_argument("--guacamole-jar", required=True)
+    parser.add_argument("--guacamole-jar", required=True,
```
Comment (ryan-williams) [inline variant_calling_benchmarks/common.py]: similar to the above, should just make this take a path to a guac directory, and have the script figure it out, no need to ship arcana about our jar-packaging through the cmdline APIs of downstream tools

File: README.md (hunk 1 above)
Comment (timodonnell) [inline README.md]: Good call, I'll do that

File: variant_calling_benchmarks/common.py (hunk 2 above)
Comment (timodonnell) [inline variant_calling_benchmarks/common.py]: Yeah, I was thinking about something like that. But then there's the quiet and error prone logic about picking which jar to use (i.e. the version stuff that `scripts/guacamole` does), which seems maybe better to just be explicit about

**Comment by timodonnell** [comment]
Updated README to address first comment, let's punt on other ways of finding or specifying jars to another PR if we decide that is worth it

**Pull request merged**